src/worknet/Commands/ContractCommand.cs

```diff
@@ -0,0 +1,39 @@
+using McMaster.Extensions.CommandLineUtils;
+using Neo;
+using Neo.BlockchainToolkit.Models;
+
+namespace NeoWorkNet.Commands
+{
+  [Command("contract", Description = "Manage neo-worknet values")]
```

Comment:
Given that there aren't any other `contract` commands, I don't think the extra command verbosity is worth it. Lets make these just `storage get`, `storage list` and `storage update`